jung-algorithms/src/main/java/edu/uci/ics/jung/layout/model/PolarPoint.java

```diff
@@ -41,17 +41,7 @@ private PolarPoint(double theta, double radius) {
     this.radius = radius;
   }
 
-  /** @return the angle for this point */
-  public double getTheta() {
-    return theta;
-  }
-
-  /** @return the radius for this point */
-  public double getRadius() {
-    return radius;
-  }
-
-  public PolarPoint setRadius(double radius) {
+  public PolarPoint newRadius(double radius) {
```

Comment:
I like this name better than `setRadius()`, but consider `withRadius()`.  (Similarly for `Point::add` -> `Point::withOffset`).

Also consider annotating this method with @CheckReturnValue, so that error-checking frameworks (e.g. errorprone) can flag calls to this method whose return value is ignored.